sql/migrations.sql

```diff
@@ -9,9 +9,10 @@ CREATE TABLE IF NOT EXISTS pg2kafka.outbound_event_queue (
   external_id   varchar(255),
   table_name    varchar(255) NOT NULL,
   statement     varchar(20) NOT NULL,
-  data          jsonb NOT NULL,
+  changed_fields varchar(64) ARRAY,
```

Comment:
why not `VARCHAR(255)` like the `table_name`?

Reply:
Good point, switched to 255 length for consistency.